### PR TITLE
Create combined translation keys file

### PR DESCRIPTION
**Pull request by VeraZab**
@bpostlethwaite @alexcjohnson for your review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,18 +13,26 @@
     "url": "https://github.com/plotly/react-plotly.js-editor/issues"
   },
   "scripts": {
+    "combine-translation-strings": "babel-node utils/combineTranslationKeys.js",
     "find-strings": "babel-node utils/findLocaleStrings.js",
```
Comment (bpostlethwaite) [inline package.json]: actually find-strings could be changed to `npm run make:translations`

Comment (bpostlethwaite) [reply package.json]: or better
`npm run make:locale` or `npm run make:localization`

File: package.json [hunk 2]
```diff
@@ -13,18 +13,26 @@
     "url": "https://github.com/plotly/react-plotly.js-editor/issues"
   },
   "scripts": {
+    "combine-translation-strings": "babel-node utils/combineTranslationKeys.js",
```
Comment (bpostlethwaite) [inline package.json]: `npm run make:combined-locales` ?

Just so it fits in with all the other make stuff?

File: package.json (hunk 1 above)
Comment (alexcjohnson) [reply package.json]: we're not making translations or localizations, we're making the list of strings from which someone would make the required translations

Comment (bpostlethwaite) [reply package.json]: or whatever. `make:something` ;)

**Comment by bpostlethwaite** [comment]
Since this 'makes' another build artifact required to run in prepublish you could give it a `make:?` name.

:dancer: other than that!

File: package.json (hunk 1 above)
Comment (bpostlethwaite) [reply package.json]: `make:translation-strings`

**Comment by VeraZab** [comment]
haha this approval button's pretty strict, another 👀  @bpostlethwaite ?

File: utils/combinedLocales.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import fs from 'fs';
 
 const pathToCombinedTranslationKeys = path.join(
   __dirname,
-  'combinedTranslationKeys.txt'
+  'combinedLocales.txt'
```
Comment (bpostlethwaite) [inline utils/combinedLocales.js]: Sorry to nitpick but as @alexcjohnson suggested perhaps we should continue calling these `translation-references` or `translation-strings` or something. Locales are the full objects with all the fancy dateFormats and such right?

Comment (VeraZab) [reply utils/combinedLocales.js]: plotly.js calls it's file translation-keys, I could call it translation-keys for the editor too, and combined-translation-keys for the combined file, works?

Comment (alexcjohnson) [reply utils/combinedLocales.js]: plotly.js has a rule of all-lowercase filenames, which seems to have fallen out of fashion in more recent js projects, but I won't vote against consistency across repos, I like your proposed names @VeraZab.

Comment (bpostlethwaite) [reply utils/combinedLocales.js]: sg!

**Comment by VeraZab** [comment]
ok, how about those filenames? and also changed some scripts around

**Pull request merged**